packages/eth-providers/vitest.config.e2e.ts

```diff
@@ -7,6 +7,7 @@ export default defineConfig({
     threads: false,
     testTimeout: 300_000,
     hookTimeout: 60_000,
+    dangerouslyIgnoreUnhandledErrors: true,
```

Comment:
Is this ignoring something known?

Reply:
Sometimes socket will throw error when we disconnect api. We had this before. I fixed some of them by adding some delay on disconnect but it's still flaky

Reply:
Maybe it's better to keep it flaky. It will remind us to fix it 😅